maven-core/src/test/java/org/apache/maven/project/artifact/MavenMetadataSourceTest.java

```diff
@@ -21,7 +21,9 @@
 
 import org.apache.maven.repository.RepositorySystem;
 import org.codehaus.plexus.PlexusTestCase;
+import org.junit.Ignore;
 
+@Ignore
```

Comment:
Why is this one skipped?

Reply:
All test methods are commented out since a73c5abc9c584 and b8c6c01864e. This change prevents the `setUp` from causing a problem, without trying to fix a test that doesn't currently run, but stops short of deleting the test.

The commenting-out is from 2009; is this a good point to drop the class?